tensorflow/lite/kernels/eigen_support.cc

```diff
@@ -130,7 +132,9 @@ RefCountedEigenContext* GetEigenContext(TfLiteContext* context) {
 }
 
 TfLiteStatus Refresh(TfLiteContext* context) {
-  SetEigenNbThreads(context->recommended_num_threads);
+  if (IsValidNumThreads(context->recommended_num_threads)) {
+    SetEigenNbThreads(context->recommended_num_threads);
```

Comment:
See my previous comment about adding this check.

Reply:
@multiverse-tf : My reply is asme as above comment.

Reply:
@multiverse-tf : Here we can not remove this check, as negative values are not permitted for eigen threads it accepts only [0, N]. However there was a sync issue, so i made some changes to improvise based on your comment. Please check. Thanks!